package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "markdown-it-bidi",
-  "version": "0.0.2",
```

Comment:
Isn't it better to change this before release?

Reply:
Since it is a patch I think you should publish it separately under the same minor version, this won't affect our plan.

Reply:
Fine